frontend/src/components/formInputs.js

```diff
@@ -28,17 +28,23 @@ export const SwitchToggle = ({
   onChange,
   labelPosition,
   small = false,
+  isDisabled = false,
 }: Object) => (
   <div className="v-mid justify-center">
     {label && labelPosition !== 'right' && <span className="di mr2 nowrap f6 dn-m">{label}</span>}
     <div className="relative dib">
       <input
-        className="absolute z-5 w-100 h-100 o-0 pointer checkbox"
+        className={`absolute z-5 w-100 h-100 o-0 ${!isDisabled && 'pointer'} checkbox`}
```

Comment:
Instead of `&&` operator, use `!isDisabled ? 'pointer' : ''`.
The `&&` operator leaves a `false` in the classname if the condition in falsy.